packages/account/src/Components/sent-email-modal/__tests__/sent-email-modal.spec.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import SentEmailModal from '../sent-email-modal';
+
+const onClose = jest.fn();
+const onClickSendEmail = jest.fn();
+
```

Comment:
I believe it will look better if these mocks on lines 6 and 7 are moved to describe block.